src/RetireNet.Packages.Tool/RetireNet.Packages.Tool.csproj

```diff
@@ -15,9 +14,12 @@
         <license>https://raw.githubusercontent.com/RetireNet/dotnet-retire/master/LICENSE</license>
         <RepositoryUrl>https://github.com/RetireNet/dotnet-retire</RepositoryUrl>
         <PackageIconUrl>https://raw.githubusercontent.com/RetireNet/images/master/nuget/dotnet-retire/icon.512x512.png</PackageIconUrl>
+        <Icon>images\icon.512x512.png</Icon>
         <RepositoryType>git</RepositoryType>
+        <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;netcoreapp2.2</TargetFrameworks>
```

Comment:
I guess this would be a breaking change, requiring clients to install .NET Core 3.1. Could be ok, but I outside the scope of the reporting feature being added. Open a new issue concerning targeting different versions of the SDK, and let's discuss it there instead of this PR.

Reply:
This was another one of those things I ended up doing to get it running within our build environment. It was building as a 3.1 project for me, which was then not running in my 2.1 build environment. This seems to let it work across all of the builds I've needed it in (we have some in 2.0, 2.1, and 2.2, no 3.0 or 3.1 yet).

It could probably be a little more complete, but I suspect this is another symptom of me not being a c# guy and using the wrong lever to do the job. :)

Reply:
I see. Yeah, might be non-breaking for clients if netcoreapp3.0 was added to the list as well, but I think even then I'd like a sep PR. There are some integration tests using Docker images that would be nice to be added as well, to test that the tool works with different SDKs / TFMs. Then the work is a bit larger and warrants a sep issue, imo. But for sure, adding more TFMs makes totally sense. I just haven't had time to setup the test for it.

Reply:
Ahh, I missed that I accidentally took out the 3.0 target, because Rider doesn't offer 3.0 as a valid target, so until I looked over the PR again just now, I thought it went from having none set (which I now realize is crazy) to having some set. Totally my bad. I'll carve it out into a separate pr. I copied the branch as is to another branch for any further work needs, and can get this one cleaned up now :)